gatsby/installing.rst

```diff
@@ -35,9 +35,11 @@ The CLI allows to initialize a project:
 This command says gatsby-cli to create a new Gatsby project with name `hello-world` with a basic default file structure.
 There are several boilerplates created by the community that allows to easily bootstrap an application for different use-cases.
 
-Boilerplates are called `starters` and in the `offical site<https://www.gatsbyjs.org/docs/gatsby-starters/>`_ you could
+These boilerplates are called `starters` and in the `offical site<https://www.gatsbyjs.org/docs/gatsby-starters/>`_ you could
 find a complete list of available starters. There are starters with some themes already configured (for example material-ui or bootstrap),
-others with the support for authentication or for some cms integration.
+others with the support for authentication or for some cms integration. There are also source-plugin specific starters which
+specifically implement and use a recommended configuration for the source-plugin, allowing users to use it to kickstart
+GatsbyJS sites with that plugin or use it as reference.
```

Comment:
The wording of this sentence is really awkward, and I don't understand what is trying to be conveyed. Can you make it more clear? I think my problem is that I don't know what is a "source-plugin".